src/Components/InstanceTypesSelect/index.js

```diff
@@ -132,7 +144,8 @@ const InstanceTypesSelect = ({ setValidation, architecture }) => {
         selections={chosenInstanceType}
         onToggle={onToggle}
         onSelect={onSelect}
-        onFilter={onFilter}
+        onFilter={() => {}}
```

Comment:
According to pf docs, `onFilter` returns the actual filtered options, and it is triggered multiple times, making it a bad practice for state changes.

`onTypeaheadInputChanged` is triggered only after a change in the typeahead input, with throttling it optimized and works smoothly, due to the implementation of the PF select, we still need an empty `onFilter` callback

Reply:
I'm missing the point why have we choose the `onTypeaheadInputChanged` instead of `onFilter`? :thinking:
Could you provide more details (maybe even in the PR description for better visibility)?